WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/CouponExpiryDateView.swift

```diff
@@ -16,31 +18,59 @@ struct CouponExpiryDateView: View {
                     DatePicker("Date picker", selection: $date, displayedComponents: .date)
                         .environment(\.timeZone, timezone)
                         .datePickerStyle(GraphicalDatePickerStyle())
-                        .onChange(of: date) { newDate in
-                            completion(newDate)
-                        }
+                        .onChange(of: date, perform: { newDate in
+                                isRemovalEnabled = true
+                            })
                     Spacer()
+                    VStack {
+                        Divider()
+                        Button(action: {
+                            onCompletion(nil)
+                            presentationMode.wrappedValue.dismiss()
+                        }, label: { Text(Localization.removeButton) })
```

Comment:
We should probably make the title bold for the button to stand out more - according to the revised design on Figma:

<img width="372" alt="Screen Shot 2022-05-27 at 10 01 05" src="https://user-images.githubusercontent.com/5533851/170620614-aab409b8-3bc2-4989-8102-9d6bb8fa444c.png">

Reply:
Much better indeed! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/6953/commits/c1d1794fba8a1cff3aa71ac0e2e456dfc5a17c8d

Reply:
@ThomazFB  - another thing that I missed: should we also add the 32-point padding at the top of the date picker view too?

Reply:
Great eye! Totally missed this spacing on top of the Date picker! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/6953/commits/4a260af44883da422a43e0039f705fb8f0cd4366